docs/sources/clients/promtail/scraping.md

```diff
@@ -53,9 +53,9 @@ There are different types of labels present in Promtail:
 - The `__path__` label is a special label which Promtail uses after discovery to
   figure out where the file to read is located. Wildcards are allowed, for example `/var/log/*.log` to get all files with a `log` extension in the specified directory, and `/var/log/**/*.log` for matching files and directories recursively. For a full list of options check out the docs for the [library](https://github.com/bmatcuk/doublestar) Promtail uses.
 
-- The `__path_exclude__` label is another special label Promtail uses after 
-  discovery, to exclude a subset of the files discovered using `__path__` from 
-  being read in the current scrape_config block. It uses the same 
+- The `__path_exclude__` label is another special label Promtail uses after
```

Comment:
nit: there's a bunch of extra-spaces deleted that adds unnecessary diffs, can we remove them?

Reply:
Yeah sorry that was my editor, Ill fix that